src/js/localStorage/savedVideos.js

```diff
@@ -0,0 +1,49 @@
+import { ALERT_MESSAGE, MAX_SAVE_COUNT } from '../constant';
+import RootLocalStorage from './rootLocalStorage';
+
+export default class SavedVideos extends RootLocalStorage {
+  constructor(key) {
+    super(key);
+  }
+
+  save(data) {
+    if (this.cached.length >= MAX_SAVE_COUNT) {
+      throw new Error(ALERT_MESSAGE.EXCEED_MAX_SAVE_VOLUME);
+    }
+    this.cache(data);
+    localStorage.setItem(this.key, JSON.stringify(data));
+  }
+
+  saveVideo(video) {
+    this.save([...this.cached, { ...video, saved: true, watched: false }]);
+  }
+
+  getWatchedVideo() {
+    return this.cached.filter(video => video.watched === true);
```

Comment:
```suggestion
    return this.cached.filter(video => video.watched);
```
심플하게 작성해도 괜찮을것 같아요~!

Reply:
실수가 많네욤.. ㅎㅎ 
반영했습니다!